packages/@ngtools/webpack/src/lazy_routes.ts

```diff
@@ -42,7 +42,7 @@ export function findLazyRoutes(
     );
   }
   if (!sourceFile) {
-    throw new Error(`Source file not found: '${fileName}'.`);
+    return {};
```

Comment:
Shouldn't the error still be thrown here?

Reply:
Re-added error.